webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ContentViewDetailsCard.js

```diff
@@ -95,7 +95,103 @@ ContentViewEnvironmentDisplay.propTypes = {
   }).isRequired,
 };
 
-const HostContentViewDetails = ({
+export const CVEDetailsBareCard = ({
+  contentViewEnvironments, hostPermissions, permissions, dropdownItems,
+  isDropdownOpen, toggleHamburger,
+}) => {
+  const userPermissions = { ...hostPermissions, ...permissions };
+  const showKebab = hasRequiredPermissions(requiredPermissions, userPermissions);
+
+  return (
+    <Card ouiaId="content-view-details-card">
+      <CardHeader>
+        <Flex
+          alignItems={{ default: 'alignItemsCenter' }}
+          justifyContent={{ default: 'justifyContentSpaceBetween' }}
+          style={{ width: '100%' }}
+        >
+          <FlexItem>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <FlexItem>
+                <CardTitle>
+                  <FormattedMessage
+                    id="cv-card-title"
+                    defaultMessage="{count, plural, =0 {Content view environments} one {Content view environment} other {Content view environments}}"
+                    values={{
+                      count: contentViewEnvironments.length,
+                    }}
+                  />
+                </CardTitle>
+              </FlexItem>
+            </Flex>
+          </FlexItem>
+          {showKebab && dropdownItems && (
+            <FlexItem>
+              <Dropdown
+                toggle={<KebabToggle aria-label="change_content_view_hamburger" onToggle={toggleHamburger} />}
+                isOpen={isDropdownOpen}
+                isPlain
+                ouiaId="change-host-content-view-kebab"
```

Comment:
should this be something more generic in AK is also using it?

Reply:
Yeah, probably. But @sambible will it be better for you if we keep it the same?

Reply:
Would be nice to see the same component share the same ouia-id across pages though, so maybe a generic ouia id in this case would be useful.

Reply:
Updated :+1: